infra/concourse/pipelines/terraform-google-vm.yml

```diff
@@ -0,0 +1,133 @@
+resource_types:
+
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+resources:
+  - name: pull-request
+    type: pull-request
+    webhook_token: ((vm.github_webhook_token))
+    source:
+      repository: terraform-google-modules/terraform-google-vm
+      access_token: ((github.pr-access-token))
+
+  - name: lint-test-image
+    type: docker-image
+    source:
+      repository: gcr.io/cloud-foundation-cicd/cft/lint
+      username: _json_key
+      password: ((sa.google))
+
+  - name: integration-test-image
+    type: docker-image
+    source:
+      repository: gcr.io/cloud-foundation-cicd/cft/kitchen-terraform
+      tag: 1.2.0
+      username: _json_key
+      password: ((sa.google))
+
+jobs:
+
+  - name: lint-tests
+    public: true
+    plan:
+      - get: pull-request
+        trigger: true
+        version: every
+      - put: notify-lint-test-pending
+        resource: pull-request
+        params:
+          path: pull-request
+          context: lint-tests
+          status: pending
+      - get: lint-test-image
+        trigger: true
+      - task: run
+        image: lint-test-image
+        config:
+          platform: linux
+          inputs:
+            - name: pull-request
+              path: terraform-google-jenkins
```

Comment:
```suggestion
              path: terraform-google-vm
```

Reply:
Aaron, thank you for your review. I've fixed my errors.